src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQualityUnitTest.java

```diff
@@ -188,4 +190,46 @@ public void testLikelihoodsEmpty_AS() throws Exception {
         Assert.assertEquals(split[0], String.format("%.2f", 0.0));
         Assert.assertEquals(split[1], String.format("%.2f", 0.0));
     }
+
+    @Test
+    public void testFinalizeRawMQ(){
+        final VariantContext vc = new VariantContextBuilder(makeVC())
+                .attribute(GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY, "2000")
+                .attribute(VCFConstants.DEPTH_KEY, 20)
+                .make();
+        final VariantContext output = new RMSMappingQuality().finalizeRawMQ(vc);
+        Assert.assertFalse(output.hasAttribute( GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY));
+        Assert.assertTrue(output.hasAttribute(VCFConstants.RMS_MAPPING_QUALITY_KEY));
+        Assert.assertEquals(output.getAttributeAsDouble(VCFConstants.RMS_MAPPING_QUALITY_KEY, -1.0), 10.0, 0.01);
+    }
+
+
+    @Test(expectedExceptions = UserException.BadInput.class)
+    public void testBadRawMQ(){
+        final VariantContext vc = new VariantContextBuilder(makeVC())
+                .attribute(GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY, "fiftyeight")
+                .make();
+        new RMSMappingQuality().finalizeRawMQ(vc);
+    }
+
+    @Test(expectedExceptions = UserException.BadInput.class)
+    public void testNoDepth(){
+        final VariantContext vc = new VariantContextBuilder(makeVC())
+                .attribute(GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY, "2000")
+                .make();
+        new RMSMappingQuality().finalizeRawMQ(vc);
+    }
+
+    @Test
+    public void testNoRawMQ(){
+        final String OTHER_KEY = "SOME_OTHER_ANNOTATION";
+        final String VALUE = "value";
+        final VariantContext vc = new VariantContextBuilder(makeVC())
+                    .attribute(OTHER_KEY, VALUE)
+                    .make();
+        final VariantContext output = new RMSMappingQuality().finalizeRawMQ(vc);
+        Assert.assertFalse(output.hasAttribute(VCFConstants.RMS_MAPPING_QUALITY_KEY));
+        Assert.assertFalse(output.hasAttribute(GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_KEY));
+        Assert.assertEquals(output.getAttribute(OTHER_KEY), VALUE);
+    }
```

Comment:
What do you want to test with the comma?  That it successfully ignores multiple values?  Or should we change that to a crash since it seems like garbage?